buildlib/azure-pipelines.yml

```diff
@@ -10,33 +10,43 @@ resources:
     - container: centos7
       image: ucfconsort.azurecr.io/ucx/centos7:1
       endpoint: ucfconsort_registry
+    - container: fedora
+      image: ucfconsort.azurecr.io/ucx/fedora:30
+      endpoint: ucfconsort_registry
 
 stages:
-  - stage: Build
+  - stage: LatestCCs
     jobs:
-      - job: Compile
-        displayName: Compile Tests
-        pool:
-          vmImage: 'Ubuntu-16.04'
+      - job: latest_cc
+        displayName: Latest CCs and CppCheck
+        container: fedora
         steps:
-          - bash: |
-              ./autogen.sh
+          - bash: ./autogen.sh
             displayName: Setup autotools
 
-          - bash: |
-              mkdir build && cd build
-              ../configure --disable-numa
+          - bash: ./contrib/configure-release
             displayName: Configure
 
+          # this will also generate compile_commands.json for cppcheck
+          - bash: bear make -j`nproc` V=1
```

Comment:
I talked to Yossi once that you guys should look at cmake.. no need for bear that way.